### PR TITLE
[6.1.0] Add missing apple docs

### PR DESCRIPTION
**Pull request by divyaamunugama**
Related issue: https://github.com/wso2/product-is/issues/16627

### CODE REVIEW

File: en/docs/guides/identity-federation/apple.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+4. WSO2 Identity Server sends the authorization code to the Apple Sign-In token endpoint and requests an ID token in return.
+5. Apple validates the client application's client ID, client secret, and authorization code and responds with an ID token, access token, and refresh token.
+6. WSO2 Identity Server validates the received ID token and extracts the authenticated user information.
+7. WSO2 Identity Server completes the authentication flow by redirecting back to the client application with the authenticated user information.
+
+![High level diagram for Apple and IS integration]({{base_path}}/assets/img/guides/apple-wso2-is-integration.png)
+
+
+## Prerequisites
+Before adding Apple as a federated authenticator on the WSO2 Identity server, you need to register an application on the Apple Developer Portal and make sure you obtain the following:
```
Comment (ThaminduDilshan) [inline en/docs/guides/identity-federation/apple.md:24]:
```suggestion
Before adding Apple as a federated authenticator on the WSO2 Identity Server, you need to register an application on the Apple Developer Portal and make sure you obtain the following:
```

File: en/docs/guides/identity-federation/apple.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+- **Client ID**: This identifies the requests sent from WSO2 Identity Server to the application.
+- **Client secret**: This authenticates the requests sent from WSO2 Identity Server to the application.
```
Comment (ThaminduDilshan) [inline en/docs/guides/identity-federation/apple.md:27]:
```suggestion
- **Client Secret**: This authenticates the requests sent from WSO2 Identity Server to the application.
```

File: en/docs/guides/identity-federation/apple.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            </tr>
+            <tr>
+                <td>OpenID Connect User ID Location</td>
+                <td>This is not relevant for this feature.</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Additional Query Parameters</td>
+                <td>These are the additional query parameters that are necessary to authenticate users through Apple Sign In.</td>
+                <td><code>scope=openid email</code></td>
```
Comment (ThaminduDilshan) [inline en/docs/guides/identity-federation/apple.md:104]:
```suggestion
                <td><code>scope=openid email&response_mode=form_post</code></td>
```
When requesting email or name scopes, apple mandates sending `response_mode=form_post` query param

File: en/docs/guides/identity-federation/apple.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+            </tr>
+            <tr>
+                <td>Token Endpoint URL</td>
+                <td>This is the token endpoint URL of Apple Sign In.</td>
+                <td><code>https://appleid.apple.com/auth/token</code></td>
+            </tr>
+            <tr>
+                <td>Callback URL</td>
+                <td>This is the service provider's URL to which authorization codes are sent. Upon successful authentication, the browser should be redirected to this URL. The URL should be specified in the following format: <code>https://:/acs</code></td>
+                <td><code>https://localhost/callback</code></td>
```
Comment (ThaminduDilshan) [inline en/docs/guides/identity-federation/apple.md:89]:
```suggestion
                <td><code>https://localhost:9443/commonauth</code></td>
```

File: en/docs/guides/identity-federation/apple.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+                <td><code>https://appleid.apple.com/auth/authorize</code></td>
+            </tr>
+            <tr>
+                <td>Token Endpoint URL</td>
+                <td>This is the token endpoint URL of Apple Sign In.</td>
+                <td><code>https://appleid.apple.com/auth/token</code></td>
+            </tr>
+            <tr>
+                <td>Callback URL</td>
+                <td>This is the service provider's URL to which authorization codes are sent. Upon successful authentication, the browser should be redirected to this URL. The URL should be specified in the following format: <code>https://:/acs</code></td>
```
Comment (ThaminduDilshan) [inline en/docs/guides/identity-federation/apple.md:88]: Shall we change this to match with other federated IDPs?
https://is.docs.wso2.com/en/latest/guides/identity-federation/facebook/#:~:text=This%20is%20the%20URL%20to%20which%20the%20browser%20should%20be%20redirected%20after%20the%20authentication%20is%20successful.%20This%20should%20be%20the%20commonauth%20endpoint%20of%20Identity%20server.

Comment (divyaamunugama) [reply en/docs/guides/identity-federation/apple.md:88]: ack will update this

**Review by ThaminduDilshan — APPROVED** [review]

**Pull request merged**